### PR TITLE
adding spring and devise gems

### PR DESCRIPTION
**Pull request by epartakki**
make sure to `bundle install` before next `rackup`

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -68,6 +68,12 @@ GEM
     byebug (11.1.3)
     concurrent-ruby (1.1.9)
     crass (1.0.6)
+    devise (4.8.1)
```
Comment (sunniraleigh) [inline Gemfile.lock:71]: what is devise for?

File: Gemfile [hunk 2]
```diff
@@ -48,7 +48,7 @@ group :development do
   # gem 'rack-mini-profiler', '~> 2.0'
   gem 'listen', '~> 3.3'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  # gem 'spring'
+  gem 'spring'
```
Comment (sunniraleigh) [inline Gemfile:51]: what is spring? why did you uncoment?

File: Gemfile [hunk 3]
```diff
@@ -71,7 +71,7 @@ gem 'dotenv-rails'
 gem 'leaflet-rails'
```
Comment (sunniraleigh) [inline Gemfile:71]: nice! exciting!

**Review by sunniraleigh — APPROVED** [review]

**Review by kvangreen — APPROVED** [review]
Looks good to me!

File: Gemfile.lock (hunk 1 above)
Comment (epartakki) [reply Gemfile.lock:71]: https://rubygems.org/gems/devise

File: Gemfile (hunk 2 above)
Comment (epartakki) [reply Gemfile:51]: Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring

**Review by luciebrock — APPROVED** [review]
love itttttt

**Pull request merged**